### PR TITLE
Added translated docs to czech

### PR DESCRIPTION
**Pull request by zen0bit**

### CODE REVIEW

File: docs/ANDROID.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+| Updated 26.02.2023 | Languages: EN, [CZ](/docs/lang/cs/ANDROID.md) |
```
Comment (epoberezkin) [inline docs/ANDROID.md:1]: Please don't change the update dates if it's the only change - it'll make other translations look older than they actually are.

File: docs/lang/cs/ANDROID.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+| Aktualizováno 26.02.2023 | Jazyky: CZ, [EN](/docs/ANDROID.md) |
```
Comment (epoberezkin) [inline docs/lang/cs/ANDROID.md:1]: I suggest to make this date match the date of the original document that you translated.

**Comment by epoberezkin** [comment]
Thanks a lot! I've made couple comments about the dates, but I can update myself of course.

File: docs/lang/cs/ANDROID.md (hunk 2 above)
Comment (zen0bit) [reply docs/lang/cs/ANDROID.md]: Wasn't in original file so...

07.02 from last commit on file is good?

**Comment by zen0bit** [comment]
No should be as requested...

**Comment by epoberezkin** [comment]
Thank you!

**Comment by epoberezkin** [comment]
Merged, but looks like image URLs need to be amended to point to the original locations... If you are able to do it, please use relative links, so it works correctly when we get around to porting all these pages to the website.

**Pull request merged**